src/Controls/src/Core/VisualElement.cs

```diff
@@ -799,7 +800,14 @@ public ResourceDictionary Resources
 		public bool Focus()
 		{
 			if (IsFocused)
+			{
+#if ANDROID
+				// TODO: Refactor using mappers for .NET 8
```

Comment:
After chatting with @PureWeen, we've agreed to do this in a follow-up PR to keep this PR backport-friendly